src/components/10-atoms/radio/index.js

```diff
@@ -25,8 +26,8 @@ class AXARadio extends NoShadowDOM {
 
   static get properties() {
     return {
-      refId: { type: String },
-      value: { type: String },
+      refId: { type: String, defaultValue: createRefId('axa-radio-') },
```

Comment:
would do it like in other form elements: defaultValue: \`radio-${createRefId()}\`

Reply:
@NoahLiechti agree but is not part of this PR. This PR is only scoped on: Apply defaults. I created an Issue for this: #1531